src/main/java/org/opentripplanner/graph_builder/module/islandpruning/PruneIslands.java

```diff
@@ -257,7 +257,7 @@ private int processIslands(
           TransitStopVertex v = (TransitStopVertex) vIter.next();
           Set<TransitMode> modes = v.getModes();
```

Comment:
Please move this code into `Subgraph` and write documentation about all the cases, in particular the case of the empty modes.

Reply:
That would also make writing a test for it easier, since you only need to test `Subgraph` not the entire module..